### DISCUSSION TITLE
Markdown to pdf: How to get a page break after a headline?

### DISCUSSION THREAD

**Discussion by merlinuwe** [jgm/pandoc#9773] [Q&A] [answered]
I have this header in markdown

```md

#### 1.1.1 Image from the web

In pdf export, this sentences doesn't reside *under* its heading.

```

When I make a pdf from it, I use this (from Obsidian enhancing export plugin):

`-V geometry:"top=2cm, bottom=1.5cm, left=2cm, right=1in",a4paper -f ${fromFormat} --resource-path="${currentDir}" --resource-path="${attachmentFolderPath}" --lua-filter="${luaDir}/linebreaks.lua" --lua-filter="${luaDir}/pdf.lua" ${ options.textemplate ? `--resource-path="${pluginDir}/textemplate" --template="${options.textemplate}"` : ` ` } -o "${outputPath}" -t pdf`

with this

`--pdf-engine=pdflatex`

This is the content of linebreaks.lua

```
--- Transform a raw HTML element which contains only a `<br>`
-- into a format-indepentent line break.
function RawInline (el)
  if el.format:match '^html' and el.text:match '%<br ?/?%>' then
    return pandoc.LineBreak()
  end
end

```
But I get this in the pdf file:

![image](https://github.com/jgm/pandoc/assets/4138869/57694077-2c1c-46e9-813c-ee33802a53ad)

What can I do to get the text that follows a header into the next line?

**Comment by jgm** [comment, marked as answer]
See https://pandoc.org/MANUAL.html#layout
In metadata set
```
block-headings: true
```

**Reply by jgm** [reply]
And don't mess around with line breaks; that's not the way to achieve it.

**Reply by merlinuwe** [reply]
Thank you very much, it works.
